dp_creator_ii/__init__.py

```diff
@@ -1,21 +1,29 @@
 """DP Creator II makes it easier to get started with Differential Privacy."""
 
-import os
 from pathlib import Path
-from argparse import ArgumentParser
+from argparse import ArgumentParser, ArgumentTypeError
 
 import shiny
 
 
 __version__ = "0.0.1"
 
 
+def existing_csv(arg):
+    path = Path(arg)
+    if not path.exists():
+        raise ArgumentTypeError(f"No such file: {arg}")
+    if path.suffix != ".csv":
```

Comment:
Is there another way to test for type == csv?  One that examines the content of the file, rather than looking for the file extension?